templates/content/node--oe-sc-event--full.html.twig

```diff
@@ -45,10 +45,14 @@
 {% endset %}
```

Comment:
right now we produce an empty 'description_list' which adds unwanted white space if 'description_items' is empty, this was no problem before, but since we made this change, we need to check description_items is not empty now before populating inpage_navigation_fields

Reply:
this was a mistake actrualy on my side, inpage_navigation_fields should not depend on description_items